manifests/8/8212369/WPR/α0.01/8212369.WPR.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: 8212369.WPR
+PackageVersion: α0.01
+PackageLocale: en-US
+Publisher: 8212369
+PackageName: WPR
+License: Unknown

```

```suggestion
License: MS-PL
```
https://github.com/8212369/WPR/blob/master/UI/WPR.UI/Pages/AboutPage.axaml